ci/testflight/galoy/testflight-values.yml

```diff
@@ -7,22 +7,43 @@ needFirebaseServiceAccount: false
 
 galoy:
   test_accounts:
-  - phone: "+50365055543"
-    code: "182731"
-    currency: "BTC"
-  - phone: "+50365055544"
-    code: "840172"
-    currency: "BTC"
-  - phone: "+50365055545"
-    code: "105369"
-    currency: "BTC"
-  - phone: "+59981730222" # bankowner
-    code: "824197"
-    role: "bankowner"
-    username: "bankowner"
-  - phone: "+59981730223" # dealer
-    code: "824199"
-    role: "dealer"
+    - ref: A
+      phone: "+50365055543"
+      code: "182731"
+      needUsdWallet: true
+      currency: BTC
+    - ref: B
+      phone: "+50365055544"
+      code: "840172"
+      phoneMetadataCarrierType: mobile
+      needUsdWallet: true
+      currency: BTC
+    - ref: C
+      phone: "+50365055545"
+      code: "105369"
+      title: business
+      currency: BTC
+    - ref: D
+      phone: "+59981730222" # bankowner
+      code: "824197"
+      needUsdWallet: true
+      role: bankowner
+      username: bankowner
+    - ref: E
+      phone: "+59981730223" # dealer
+      code: "824199"
+      role: dealer
+    - ref: F
```

Comment:
Original object only had 5 elements configured here, but additional 3 come from `default.yaml`. Tbd if these additional 3 are necessary for testflight.